doc/DotNet-API.rst

```diff
@@ -0,0 +1,29 @@
+.Net Framework
+==============
+
+DeepSpeech
+----------
+
+.. doxygenclass:: DeepSpeechClient::DeepSpeech
+   :project: deepspeech-dotnet
+   :members:
+
```

Comment:
@lissyx  It would be great if the interface is also included:

`.. doxygenclass:: DeepSpeechClient::IDeepSpeech`

Reply:
That's one of the thing I was unsure about : does it make sense to have both? And why?

Reply:
@lissyx just FTR for dependency injection, users will now which interface and type are available for registration.